UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -190,19 +190,20 @@ <h4>Examples</h4>
 
 					<aside class="example" title="Descriptive explanations">
 						<ul>
-							<li>All content available for computer-generated speech and braille reading.</li>
-							<li>Portions of the content will not be available in computer-generated speech and
-								braille.</li>
-							<li>Portions of the content may not be available in computer-generated speech and
-								braille.</li>
+							<li>All content is available for computer-generated speech and refreshable braille reading.</li>
+														<li>Portions of the content may not be available in computer-generated speech and
+								refreshable braille.</li>
+<li>Portions of the content is not available in computer-generated speech and
+								electronic braille.</li>
 						</ul>
 					</aside>
 
 					<aside class="example" title="Compact explanations">
 						<ul>
-							<li>Readable in computer-generated audio and braille.</li>
-							<li>Not fully readable in computer-generated audio and braille.</li>
-							<li>May not be fully readable in computer-generated audio and braille.</li>
+							<li>Readable in computer-generated audio and refreshable braille.</li>
```

Comment:
This example uses "refreshable braille" but the next two use "electronic braille". I know the note says they're interchangeable, but it would be good to be consistent within any one set of example text.

Also, should these examples say "readable as" or "readable using"? Something about "readable in" makes it sound like the publication is providing the audio or braille, but maybe it's just me.